monkey/monkey_island/cc/ui/src/components/configuration-components/CredentialsConfig.tsx

```diff
@@ -18,6 +18,7 @@ export default function CredentialsConfig(props) {
           uiSchema={uiSchema}
           formData={credentialsCopy}
           onChange={(formData) => {onChange(formData.formData)}}
+          // @ts-ignore
           customFormats={customFormats}
```

Comment:
Why do we need to do this?

Reply:
Because we're using an older version of react-jsonshema-form, which, apparently, doesn't have this type definition for form props